### DISCUSSION TITLE
k40

### DISCUSSION THREAD

**Discussion by achduue** [community/community#102631] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

1. The latest version of the system K40 will cause the system to crash when it turns on airplane mode, which eventually leads to a failure to turn on. 2. There will be 2 status bars when calling, which looks a bit strange 

thanks you🥰🥰

**Comment by queenofcorgis** [comment]
Hi @achduue ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on K40, such as [their website](https://k40.com/). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
